<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T4good_hector_dlg</class>
 <widget class="QDialog" name="T4good_hector_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Here you may define when you consider a HECTOR data&lt;br/&gt;to be a &amp;quot;good data&amp;quot; (in current event)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="textLabel1">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>&lt;font size=&quot;+1&quot;&gt;BaF crystal should deliver 3 signals:&lt;br&gt;&lt;tt&gt; a). Energy FAST, &lt;/tt&gt;    &lt;tt&gt;b). Energy  SLOW, &lt;/tt&gt;     &lt;tt&gt;c) Time&lt;/tt&gt;&lt;/font&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Raw</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="textLabel1_3">
         <property name="text">
          <string>&lt;font color=&quot;#005500&quot; size=&quot;+2&quot;&gt;Raw&lt;/font&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_2">
         <property name="maximumSize">
          <size>
           <width>32767</width>
           <height>77</height>
          </size>
         </property>
         <property name="text">
          <string>When &lt;b&gt;at least one&lt;/b&gt; of these signals comes - the crystal &quot;&lt;font color=&quot;#ff0000&quot;&gt;&lt;b&gt;fired&lt;/b&gt;&lt;/font&gt;&quot;&lt;br&gt;In this case &lt;b&gt;&lt;font color=&quot;#ff0000&quot;&gt;raw&lt;/font&gt;&lt;/b&gt; spectra are incremeted - by NON_ZERO value (Franco!) &lt;br&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignVCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>&quot;Good&quot; data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="textLabel1_3_2_2">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;font color=&quot;#005500&quot; size=&quot;+3&quot;&gt;&quot;Good&quot;&lt;/font&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textLabel1_4">
           <property name="text">
            <string>Flag &quot;Good&quot; is set when the signals have sensible values.&lt;br&gt;Only such BaF crystals contribute to the &lt;b&gt;multiplicity&lt;/b&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; font-style:italic; color:#ff0000;&quot;&gt;Shortly speaking: &lt;br /&gt;&amp;quot;good data&amp;quot; means the data which passes some of your thresholds&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>To set flag 'good' as 'true'  - you demand:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="_5">
              <item>
               <widget class="QCheckBox" name="checkBox_good_enFAST">
                <property name="text">
                 <string>the &quot;energy FAST calibrated &quot; value is bigger than:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushline_enFAST_good_threshold">
                <property name="text">
                 <string>click_here</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacer3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>110</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="_6">
              <item>
               <widget class="QCheckBox" name="checkBox_good_enSLOW">
                <property name="text">
                 <string>the &quot;energy SLOW calibrated&quot; value is bigger than:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushline_enSLOW_good_threshold">
                <property name="text">
                 <string>pushButton5</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacer4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>60</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="_7">
              <item>
               <widget class="QCheckBox" name="checkBox_good_time">
                <property name="text">
                 <string>the  &quot;Time calibrated&quot; value is bigger than:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="push_time_threshold">
                <property name="text">
                 <string>click here</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="spacer5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>178</width>
                  <height>31</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_not_led">
              <property name="text">
               <string>When signals are NOT LED  (real physics event)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Doppler</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="_21">
         <item>
          <widget class="QLabel" name="textLabel1_5">
           <property name="text">
            <string>&lt;font color=&quot;#005500&quot; size=&quot;+2&quot;&gt;Doppler correction&lt;/font&gt; is prerformed when for all &quot;good&quot; events.&lt;br&gt;
When the algorithm make doppler correction succesfully - the&lt;b&gt; OK&lt;/b&gt; flag is set.&lt;br&gt; 
&lt;i&gt;(It may not be succesfull for example when we demand correct tracking&lt;br&gt; and the multiwire chamber mw42 didn't give the position information)&lt;/i&gt; </string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>LED signals</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Here you may decide what means that the LED marker is true</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="11">
           <spacer name="spacer7_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>77</width>
              <height>31</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="6">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="10">
           <widget class="QPushButton" name="push_adc_high">
            <property name="text">
             <string>0000</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="textLabel1_6">
            <property name="text">
             <string>&lt;font size=&quot;+1&quot;&gt;LED is &quot;true&quot; when&lt;/font&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QPushButton" name="push_LED_Tdc">
            <property name="text">
             <string>TDC[00]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="10">
           <widget class="QPushButton" name="push_tdc_high">
            <property name="text">
             <string>0000</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="textLabel2_2">
            <property name="text">
             <string>OR</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QPushButton" name="push_tdc_low">
            <property name="text">
             <string>0000</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="textLabel2_2_2_2">
            <property name="text">
             <string>is in region</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="push_LED_Adc">
            <property name="text">
             <string>ADC[00]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="9">
           <widget class="QLabel" name="textLabel2_2_4">
            <property name="text">
             <string>-</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QPushButton" name="push_adc_low">
            <property name="text">
             <string>0000</string>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QLabel" name="textLabel2_2_3">
            <property name="text">
             <string>-</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="11">
           <spacer name="spacer7_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>131</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="5">
           <widget class="QLabel" name="textLabel2_2_2">
            <property name="text">
             <string>is in region</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_20">
     <item>
      <widget class="QPushButton" name="push_default">
       <property name="text">
        <string>Default settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="push_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="puss_accept">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>puss_accept</sender>
   <signal>clicked()</signal>
   <receiver>T4good_hector_dlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>push_cancel</sender>
   <signal>clicked()</signal>
   <receiver>T4good_hector_dlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>480</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
